myhpom/templates/myhpom/accounts/choose_network.html

```diff
@@ -1,9 +1,12 @@
 {% extends "myhpom/base.html" %}
 
+{% block body_on_load %}onLoad="document.getElementById('id_custom_provider').select();"{% endblock %}
```

Comment:
In keeping with the general commandment "thou shall not inline css or js" I think this needs a little refinement...like...create an explicit script tag at the bottom of the base.html template using `jQuery.noConflict()(function ($) {})` and put a block in there that other templates can tap into.

Reply:
Yeah, you're right, especially since I already have a block there....